src/wallet/wallet.h

```diff
@@ -155,6 +155,9 @@ struct COutputEntry
 /** A transaction with a merkle branch linking it to the block chain. */
 class CMerkleTx : public CTransaction
 {
+private:
+    static const uint256 abandonHash;
```

Comment:
Can you explain why this static field on CMerkleTx is necessary? (and add a comment) What is it used for, what coordinates/synchronizes access to it?
(e.g. why is there only one "abandoned transaction hash"? why is it not passed in some other way?)